### PR TITLE
Implemented stars filter + improved filtering

### PR DESCRIPTION
**Pull request by ramonvermeulen**
### Checklist

Solves https://github.com/godatadriven/github-contributions/issues/80 and https://github.com/godatadriven/github-contributions/issues/52 and https://github.com/godatadriven/github-contributions/issues/49

Already deployed it so you can play a bit around: https://godatadriven.github.io/github-contributions/

- [ ] When adding a Github handle to be tracked that is not yours, request the user to approve the PR

### CODE REVIEW

File: webapp/src/components/SelectBox.tsx [hunk 1]
```diff
@@ -27,7 +23,7 @@ function SelectBox({
             <Select
                 labelId="demo-simple-select-label"
```
Comment (JCZuurmond) [inline webapp/src/components/SelectBox.tsx:24]: Should we rename the label and id to something else than "demo-simple-..."

File: webapp/src/pages/home/Home.tsx [hunk 2]
```diff
@@ -1,49 +1,86 @@
...
+    `;
 
-    const authorQuery = 'SELECT distinct author FROM main_marts.fct_pull_requests ORDER BY lower(author);';
-    const organizationQuery = 'SELECT distinct author_organization AS organization FROM main_marts.fct_pull_requests ORDER BY lower(author_organization);';
-    const repositoryQuery = 'SELECT distinct repository FROM main_marts.fct_pull_requests ORDER BY lower(repository);';
-    const ownerQuery = 'SELECT distinct owner FROM main_marts.fct_pull_requests ORDER BY lower(owner);';
-    const pullRequestCountQuery = `SELECT count(*) as amount FROM main_marts.fct_pull_requests ${useQueryFilter(filters)};`;
-    const repoCountQuery = `SELECT count(distinct repository) as amount FROM main_marts.fct_pull_requests ${useQueryFilter(filters)};`;
-    const contributorCountQuery = `SELECT count(distinct author) as amount FROM main_marts.fct_pull_requests ${useQueryFilter(filters)};`;
+    const organizationQuery = 'SELECT distinct pr.author_organization AS organization FROM main_marts.fct_pull_requests pr LEFT JOIN main_marts.fct_repositories rep ON pr.full_repository_name = rep.full_name ORDER BY lower(pr.author_organization);';
```
Comment (JCZuurmond) [inline webapp/src/pages/home/Home.tsx]: These are starting to get big; we should think what we want to do with this

**Review by JCZuurmond — DISMISSED** [review]
@ramonvermeulen : Awesome! This is great, makes the dashboard more useful by filtering relevant projects.

A bonus would be to update the drop down lists when the star filter changes. Only allow to filter on repo's, authors, organzations or owners that have (contributed to) repo's with more stars than the filter

File: webapp/src/pages/home/Home.tsx (hunk 2 above)
Comment (ramonvermeulen) [reply webapp/src/pages/home/Home.tsx]: I do agree, created another issue for this: https://github.com/godatadriven/github-contributions/issues/82

Ideally I would like to have some kind of ORM implemented. Currently SQL queries and filtering logic is getting quite big. So I will look into ways to clean this a bit.

**Pull request merged**